src/panels/lovelace/editor/types.ts

```diff
@@ -19,6 +20,12 @@ export interface ViewEditEvent extends Event {
   };
 }
 
+export interface ViewVisibilityChangeEvent extends Event {
+  detail: {
+    visible: ShowViewConfig[];
+  };
+}
```

Comment:
```suggestion
export interface ViewVisibilityChangeEvent {
    visible: ShowViewConfig[];
}
```
and use `HASSDomEvent<ViewVisibilityChangeEvent>` instead

Reply:
Also use `ViewVisibilityChangeEvent` when you declare the event. And move this there.
https://github.com/home-assistant/frontend/pull/4978/files#diff-33e1deebcb55da6fbd461e09dbe65d67R25

Reply:
what about the other two? `ViewEditEvent` and `EntitiesEditorEvent` should I also update them and use `HASSDomEvent`?

Reply:
done. I'll leave the rest because the `EntitiesEditorEvent` is used in 45 places. Don't want to mess anything.